sdk/eventhub/event-hubs/samples/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "azure-event-hubs-samples",
+  "version": "1.0.0",
+  "description": "The samples in this folder are for version 5.0.0 and above of this library. If you are using version 2.1.0 or lower, then please use [samples for v2.1.0](https://github.com/Azure/azure-sdk-for-js/tree/%40azure/event-hubs_2.1.0/sdk/eventhub/event-hubs/samples) instead",
+  "main": "dist/runsamples.js",
+  "scripts": {
+    "samples": "ts-node ./runsamples.ts",
+    "build": "tsc -p ."
+  },
+  "keywords": [
+    "Azure",
+    "Storage",
+    "File",
+    "Node.js",
+    "TypeScript"
+  ],
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+  "homepage": "https://github.com/Azure/azure-sdk-for-js#readme",
+  "devDependencies": {
+    "ts-node": "^8.5.4",
+    "typescript": "^3.7.2"
+  },
+  "dependencies": {
+    "@azure/core-amqp": "next",
```

Comment:
We'll have to remember when we do the GA release to turn these back to the GA'd versions :)

Reply:
Yes, or perhaps 'latest'?

Reply:
I'd actually prefer `^5.0.0` since that will get the 'latest' version that's API compatible with the current samples. I think `next` is ok for now since we're not following a typical semver scheme while doing the preview releases.